ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -17,6 +17,14 @@
 ########################
 
 ### Single Zone Units ###
+AHU_SS_SDM:
```

Comment:
This is not a complete type. For AHU, we expect fan types, cooling/heating types, economizer types, etc.

Reply:
@cstirdivant - thanks for your response. it is a heat recovery unit - It has the following points from the BMS commissioning engineers - 
run_command
supply_air_damper_command
run_command
. Can you provide me with any guidance. Thank you

Reply:
@ccquigley - apologies have received updated info from commissioning engineers. I have remodelled this as a FCU. It is a standalone unit that exchanges air using the damper.